### ISSUE TITLE
Error /f disband

### ISSUE THREAD

**Issue by ghost** [magnusulf/Factions#9]
## Affected Plugin

- **Affected Plugin Name:** Factions
- **Affected Plugin Version:** 1.3.2

## Describe your Environment

- **Server Version:** 1.13.2

## Steps to Reproduce this Bug

Does not work `/f disband`. Faction does not disband even when there are no players.

**Comment by magnusulf** [comment]
Have you noticed that you have to type in a confirmation text, to make sure disbanding is not done accidentally? 

It seems to work for me.

![screenshot 2019-02-10 at 17 08 23](https://user-images.githubusercontent.com/6261088/52536121-87c85000-2d56-11e9-8dd6-31aa82daec3f.png)

**Comment by Uranusak** [comment]
Error from console 
```
[18:46:02 INFO]: Uran234 issued server command: /f disband test
[18:46:10 INFO]: Uran234 issued server command: /f disband test 4PalTU
[18:46:10 INFO]: [Factions 3.1.2] The faction test (a77c8c4d-997f-419d-b5b3-fc2e22c8c641) was disbanded by Uran234.
[18:46:10 ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing 'f disband test 4PalTU' in com.massivecraft.massivecore.command.MassiveCoreBukkitCommand(f)
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:146) ~[spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
        at org.bukkit.craftbukkit.v1_13_R2.CraftServer.dispatchCommand(CraftServer.java:704) ~[spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
        at net.minecraft.server.v1_13_R2.PlayerConnection.handleCommand(PlayerConnection.java:1636) ~[spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
        at net.minecraft.server.v1_13_R2.PlayerConnection.a(PlayerConnection.java:1476) ~[spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
        at net.minecraft.server.v1_13_R2.PacketPlayInChat.a(PacketPlayInChat.java:45) ~[spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
        at net.minecraft.server.v1_13_R2.PacketPlayInChat.a(PacketPlayInChat.java:1) ~[spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
        at net.minecraft.server.v1_13_R2.PlayerConnectionUtils.lambda$0(PlayerConnectionUtils.java:9) ~[spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_191]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_191]
        at net.minecraft.server.v1_13_R2.SystemUtils.a(SourceFile:199) [spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
        at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:896) [spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
        at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:417) [spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
        at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:831) [spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
        at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:729) [spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_191]
Caused by: java.lang.NullPointerException: senderObject
        at com.massivecraft.massivecore.util.IdUtil.getOfflinePlayer(IdUtil.java:783) ~[?:?]
        at com.massivecraft.massivecore.money.MoneyMixinVault.getEconomyObject(MoneyMixinVault.java:258) ~[?:?]
        at com.massivecraft.massivecore.money.MoneyMixinVault.move(MoneyMixinVault.java:188) ~[?:?]
        at com.massivecraft.massivecore.money.MoneyMixinAbstract.move(MoneyMixinAbstract.java:64) ~[?:?]
        at com.massivecraft.massivecore.money.MoneyMixinAbstract.set(MoneyMixinAbstract.java:139) ~[?:?]
        at com.massivecraft.massivecore.money.Money.set(Money.java:236) ~[?:?]
        at com.massivecraft.factions.entity.Faction.preDetach(Faction.java:97) ~[?:?]
        at com.massivecraft.massivecore.store.EntityContainerAbstract.preDetach(EntityContainerAbstract.java:354) ~[?:?]
        at com.massivecraft.massivecore.store.EntityContainerAbstract.detachFixed(EntityContainerAbstract.java:327) ~[?:?]
        at com.massivecraft.massivecore.store.EntityContainerAbstract.detachEntity(EntityContainerAbstract.java:305) ~[?:?]
        at com.massivecraft.massivecore.store.EntityContainerAbstract.detachEntity(EntityContainerAbstract.java:16) ~[?:?]
        at com.massivecraft.massivecore.store.EntityInternal.detach(EntityInternal.java:83) ~[?:?]
        at com.massivecraft.factions.cmd.CmdFactionsDisband.perform(CmdFactionsDisband.java:87) ~[?:?]
        at com.massivecraft.massivecore.command.MassiveCommand.execute(MassiveCommand.java:1177) ~[?:?]
        at com.massivecraft.massivecore.command.MassiveCommand.execute(MassiveCommand.java:1133) ~[?:?]
        at com.massivecraft.massivecore.command.MassiveCoreBukkitCommand.execute(MassiveCoreBukkitCommand.java:60) ~[?:?]
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:139) ~[spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
        ... 14 more
```

**Comment by ghost** [comment]
If I leave the faction, the faction will not be removed. Then, the faction can not be removed with / f disband.

```
[18:53:46 INFO]: Uran234 issued server command: /f create test3
[18:53:46 INFO]: [Factions 3.1.2] Uran234 created a new faction: test3
[18:53:51 INFO]: Uran234 issued server command: /f leave
[18:53:51 INFO]: [Factions 3.1.2] Uran234 left the faction: test3
[18:53:51 INFO]: [Factions 3.1.2] The faction test3 (738b2c94-033e-4fad-ba44-ac048de784f3) was disbanded due to the last player (Uran234) leaving.
[18:53:51 ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing 'f leave' in com.massivecraft.massivecore.command.MassiveCoreBukkitCommand(f)
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:146) ~[spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
        at org.bukkit.craftbukkit.v1_13_R2.CraftServer.dispatchCommand(CraftServer.java:704) ~[spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
        at net.minecraft.server.v1_13_R2.PlayerConnection.handleCommand(PlayerConnection.java:1636) ~[spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
        at net.minecraft.server.v1_13_R2.PlayerConnection.a(PlayerConnection.java:1476) ~[spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
        at net.minecraft.server.v1_13_R2.PacketPlayInChat.a(PacketPlayInChat.java:45) ~[spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
        at net.minecraft.server.v1_13_R2.PacketPlayInChat.a(PacketPlayInChat.java:1) ~[spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
        at net.minecraft.server.v1_13_R2.PlayerConnectionUtils.lambda$0(PlayerConnectionUtils.java:9) ~[spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_191]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_191]
        at net.minecraft.server.v1_13_R2.SystemUtils.a(SourceFile:199) [spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
        at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:896) [spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
        at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:417) [spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
        at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:831) [spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
        at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:729) [spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_191]
Caused by: java.lang.NullPointerException: senderObject
        at com.massivecraft.massivecore.util.IdUtil.getOfflinePlayer(IdUtil.java:783) ~[?:?]
        at com.massivecraft.massivecore.money.MoneyMixinVault.getEconomyObject(MoneyMixinVault.java:258) ~[?:?]
        at com.massivecraft.massivecore.money.MoneyMixinVault.move(MoneyMixinVault.java:188) ~[?:?]
        at com.massivecraft.massivecore.money.MoneyMixinAbstract.move(MoneyMixinAbstract.java:64) ~[?:?]
        at com.massivecraft.massivecore.money.MoneyMixinAbstract.set(MoneyMixinAbstract.java:139) ~[?:?]
        at com.massivecraft.massivecore.money.Money.set(Money.java:236) ~[?:?]
        at com.massivecraft.factions.entity.Faction.preDetach(Faction.java:97) ~[?:?]
        at com.massivecraft.massivecore.store.EntityContainerAbstract.preDetach(EntityContainerAbstract.java:354) ~[?:?]
        at com.massivecraft.massivecore.store.EntityContainerAbstract.detachFixed(EntityContainerAbstract.java:327) ~[?:?]
        at com.massivecraft.massivecore.store.EntityContainerAbstract.detachEntity(EntityContainerAbstract.java:305) ~[?:?]
        at com.massivecraft.massivecore.store.EntityContainerAbstract.detachEntity(EntityContainerAbstract.java:16) ~[?:?]
        at com.massivecraft.massivecore.store.EntityInternal.detach(EntityInternal.java:83) ~[?:?]
        at com.massivecraft.factions.entity.MPlayer.leave(MPlayer.java:781) ~[?:?]
        at com.massivecraft.factions.cmd.CmdFactionsLeave.perform(CmdFactionsLeave.java:24) ~[?:?]
        at com.massivecraft.massivecore.command.MassiveCommand.execute(MassiveCommand.java:1177) ~[?:?]
        at com.massivecraft.massivecore.command.MassiveCommand.execute(MassiveCommand.java:1133) ~[?:?]
        at com.massivecraft.massivecore.command.MassiveCoreBukkitCommand.execute(MassiveCoreBukkitCommand.java:60) ~[?:?]
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:139) ~[spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
        ... 14 more
```

```
f disband test3
[19:04:51 INFO]: Created confirmation text sw8lpv for you.
[19:04:51 INFO]: To disband faction confirm it by typing:
/f disband test3 sw8lpv
f disband test3 sw8lpv
[19:05:13 INFO]: You disbanded test3.
[19:05:13 INFO]: [Factions 3.1.2] The faction test3 (738b2c94-033e-4fad-ba44-ac048de784f3) was disbanded by @console.
[19:05:13 WARN]: Unexpected exception while parsing console command "f disband test3 sw8lpv"
org.bukkit.command.CommandException: Unhandled exception executing 'f disband test3 sw8lpv' in com.massivecraft.massivecore.command.MassiveCoreBukkitCommand(f)
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:146) ~[spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
        at org.bukkit.craftbukkit.v1_13_R2.CraftServer.dispatchCommand(CraftServer.java:704) ~[spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
        at org.bukkit.craftbukkit.v1_13_R2.CraftServer.dispatchServerCommand(CraftServer.java:689) [spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
        at net.minecraft.server.v1_13_R2.DedicatedServer.handleCommandQueue(DedicatedServer.java:459) [spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
        at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:418) [spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
        at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:831) [spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
        at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:729) [spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_191]
Caused by: java.lang.NullPointerException: senderObject
        at com.massivecraft.massivecore.util.IdUtil.getOfflinePlayer(IdUtil.java:783) ~[?:?]
        at com.massivecraft.massivecore.money.MoneyMixinVault.getEconomyObject(MoneyMixinVault.java:258) ~[?:?]
        at com.massivecraft.massivecore.money.MoneyMixinVault.move(MoneyMixinVault.java:188) ~[?:?]
        at com.massivecraft.massivecore.money.MoneyMixinAbstract.move(MoneyMixinAbstract.java:64) ~[?:?]
        at com.massivecraft.massivecore.money.MoneyMixinAbstract.set(MoneyMixinAbstract.java:139) ~[?:?]
        at com.massivecraft.massivecore.money.Money.set(Money.java:236) ~[?:?]
        at com.massivecraft.factions.entity.Faction.preDetach(Faction.java:97) ~[?:?]
        at com.massivecraft.massivecore.store.EntityContainerAbstract.preDetach(EntityContainerAbstract.java:354) ~[?:?]
        at com.massivecraft.massivecore.store.EntityContainerAbstract.detachFixed(EntityContainerAbstract.java:327) ~[?:?]
        at com.massivecraft.massivecore.store.EntityContainerAbstract.detachEntity(EntityContainerAbstract.java:305) ~[?:?]
        at com.massivecraft.massivecore.store.EntityContainerAbstract.detachEntity(EntityContainerAbstract.java:16) ~[?:?]
        at com.massivecraft.massivecore.store.EntityInternal.detach(EntityInternal.java:83) ~[?:?]
        at com.massivecraft.factions.cmd.CmdFactionsDisband.perform(CmdFactionsDisband.java:87) ~[?:?]
        at com.massivecraft.massivecore.command.MassiveCommand.execute(MassiveCommand.java:1177) ~[?:?]
        at com.massivecraft.massivecore.command.MassiveCommand.execute(MassiveCommand.java:1133) ~[?:?]
        at com.massivecraft.massivecore.command.MassiveCoreBukkitCommand.execute(MassiveCoreBukkitCommand.java:60) ~[?:?]
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:139) ~[spigot.jar:git-Spigot-b0f4c22-a2dd0e3]
        ... 7 more
```

**Comment by magnusulf** [comment]
Thank you for reporting these bugs, I think I forgot to say that earlier 😄 

I think this one as well as #7 is fixed in 3.1.3, I am still looking into the other ones.

**Comment by ghost** [comment]
v3.1.3 /f disband bug fixed
